runtime/src/main/java/org/corfudb/runtime/clients/LogUnitClient.java

```diff
@@ -33,6 +35,10 @@
     @Getter
     IClientRouter router;
 
+    public String getHost() { return "test-host"; }
```

Comment:
this is obvioulsy a left-over. 

I would like to move the host:ip prefix setup to the setup time of Runtime.metrics , in order to do it only once in the code, and remove the inspection of hostname and IP port from client-side classes.